_posts/2024-11-26-OSPREY-Wins-HPCwire-Award.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+author: Sophie Bui
+title: OSPREY Wins HPCwire Editor's Choice Award for Best HPC Response to Societal Plight
+excerpt: OSPREY, an open science platform for robust epidemic analysis that leverages Globus Compute, a unified interface for securely and reliably accessing remote compute resources using a distributed network of Parsl executors to create a Function as a Service (FaaS) platform, was awarded the Editor’s Choice Award at SC24 in Atlanta, Georgia, on November 18, 2024.
+---
+<img src="/images/blog/2024-11-14/SC24-Parsl-Banner.jpg" width="100%" style="border:0px solid black;" alt="A banner graphic with Parsl logo and SC24 logo;">
+{: style="text-align: center;"}
+
+Each year, <a href="https://www.hpcwire.com/" target="_blank">HPCwire</a> presents its annual <a href="https://www.hpcwire.com/2024-hpcwire-awards%20readers-editors-choice/" target="_blank">Readers’ and Editors’ Choice Awards</a> to celebrate the most significant breakthroughs in high-performance computing (HPC) over the past 12 months. The global HPC community nominated and voted on the awards, with the winners announced and honored during the Supercomputing Conference. These recognitions within the HPC industry honor outstanding achievements and innovations and serve as a mark of excellence in the field.
```

Comment:
"The global HPC community nominated and voted on the awards" should be "The global HPC community nominated and voted on the Readers' Choice Awards"